python/rikai/types/vision.py

```diff
@@ -209,6 +213,15 @@ def _repr_png_(self):
     def __eq__(self, other) -> bool:
         return isinstance(other, Image) and super().__eq__(other)
 
+    def __or__(self, other: Drawable) -> Draw:
+        """Override ``&`` operator to chain images with
+        visualization components.
+        """
+        return self.draw(other)
+
+    def render(self, render: "rikai.viz.Render", **kwargs) -> None:
```

Comment:
Should Image really be a Drawable?

Reply:
Good question. lets not make `Image` a `Drawable` for now.